### ISSUE TITLE
too-few-format-argument is not raised

### ISSUE THREAD

**Issue by sam-s** [PyCQA/pylint#1529]
### Steps to reproduce
```
import pandas as pd
a=1
print "%s %s" % (pd.DataFrame().a)
```
note the mistake: the period `.` between `()` and `a` should be comma `,`

### Current behavior

no error, no warning

### Expected behavior

`error (E1306, too-few-format-args, ) Not enough arguments for format string`

### pylint --version output
```
pylint 1.7.1, 
astroid 1.5.3
Python 2.7.13 (default, Dec 18 2016, 07:03:39) 
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.42.1)]
```

**Comment by PCManticore** [comment]
Thanks for reporting this issue! Makes sense.

**Comment by PCManticore** [comment]
I took a look at this issue, but in the end I'm not sure it's worth fixing it. The problem we have is that for this check, we are operating at the AST level. And in this case, we can't know if the original code used parens or not for the right hand side of the operation, so say we can't make a difference between ``string % value.a`` and ``string % (value.a)``. Because of this and because we can't know what the right hand side entails (e.g it might be a property that returns a two-element tuple or anything like that), we decided not to emit this check. A solution for solving this problem would be to get the original line and then tokenize it and look for the occurence of the string interpolation operation, in a similar way to how we are doing for ``trailing-comma-tuple``. But I find this solution too brittle for this particular check, since for the aforementioned check it was rather easier, due to the fact that it was implemented from the start as a token checker, rather than an AST check as ``too-few-format-args`` is.
